optaplanner-spring-integration/optaplanner-spring-boot-autoconfigure/src/main/java/org/optaplanner/spring/boot/autoconfigure/OptaPlannerAutoConfiguration.java

```diff
@@ -162,6 +162,12 @@ public SolverConfig solverConfig() {
                         throw new UnsupportedOperationException(noConstraintProviderErrorMsg);
                     }
 
+                    @Override
+                    public ConstraintVerifier<ConstraintProvider_, SolutionClass_> withDroolsAlphaNetworkCompiler(
+                            boolean droolsAlphaNetworkCompilerEnabled) {
+                        throw new UnsupportedOperationException(noConstraintProviderErrorMsg);
```

Comment:
(separate issue, not part of this PR) cc @Christopher-Chianelli if you could take a look, Chris:

Let's also improve the error message for clarity, so it :
```
final String noConstraintProviderErrorMsg = "ConstraintVerifier is only supported for ConstraintProviders";
```
=>
```
final String noConstraintProviderErrorMsg = "Cannot create a " + ConstraintVerifier.class.getSimpleName() + " because there is no " + ConstraintProvider.class.getSimpleName() + " configured in the " + SolverConfig.class.getSimpleName() + ".";
```